app/src/main/res/values/styles.xml

```diff
@@ -410,21 +410,23 @@
     <item name="android:textColor">@color/component_color_shared_text_view_text_color</item>
   </style>
   <!-- Start Over Lesson Button -->
-  <style name="StartOverLessonButton" parent="TextAppearance.AppCompat.Widget.Button">
+  <style name="SecondaryButton" parent="TextAppearance.AppCompat.Widget.Button">
     <item name="android:layout_width">144dp</item>
     <item name="android:layout_height">wrap_content</item>
     <item name="android:paddingEnd">12dp</item>
     <item name="android:paddingStart">12dp</item>
     <item name="android:minWidth">144dp</item>
-    <item name="android:background">@drawable/start_over_button_background</item>
+    <item name="android:background">@drawable/secondary_button_background</item>
     <item name="android:fontFamily">sans-serif-medium</item>
-    <item name="drawableStartCompat">@drawable/ic_start_over_24dp</item>
     <item name="android:drawablePadding">4dp</item>
-    <item name="drawableTint">@color/component_color_shared_resume_lesson_start_over_button_color</item>
+    <item name="drawableTint">@color/component_color_shared_secondary_button_background_color</item>
```

Comment:
As suggest in above comment we can use `color_palette_primary_button_background_color` instead of this `component_color_shared_secondary_button_background_color`